examples/seismic/acoustic/operators.py

```diff
@@ -144,10 +146,28 @@ def ForwardOperator(model, geometry, space_order=4,
 
     # Create interpolation expression for receivers
     rec_term = rec.interpolate(expr=u)
-
+    # Build operator equations
+    equations = eqn + src_term + rec_term
+
+    if factor:
+        # Implement snapshotting
+        nsnaps = (geometry.nt + factor - 1) // factor
+        time_subsampled = ConditionalDimension(
+            't_sub', parent=model.grid.time_dim, factor=factor)
+        usnaps = TimeFunction(name='usnaps', grid=model.grid,
```

Comment:
You still have `u` with full time saved line 135 you can't have both

Reply:
Removed.